index.html

```diff
@@ -456,32 +421,171 @@ <h2 id="constraints">
-          allows an application to observe whether the surface that is captured is a <a>logical
-          display surface</a>, rather than the <a>visible display surface</a>.
-        </p>
-        <p>
-          The <code><a data-link-for="MediaTrackConstraintSet">cursor</a></code> constraint
-          allows an application to specify if the cursor should be included in the captured
-          display.
-        </p>
+        <p>Constraints serve a different purpose for display device capture. They
+        do not aid discovery, instead they are applied only after user-selection.
+        Some of these constraints enable user agent processing like downscaling
+        and frame decimation, as well as display-specific features. Others enable
+        observation of inherent properties of a user-selected
+        <a>display surface</a>, as capabilities and settings.</p>
+        <p>The following new and existing <code><a>MediaStreamTrack</a></code>
+        <a href="https://www.w3.org/TR/mediacapture-streams/#constrainable-properties">
+        Constrainable Properties</a> are defined to apply to the user-selected
+        video <a>display surface</a>, with the following behavior:</p>
+        <table class="simple">
+          <thead>
+            <tr>
+              <th>Property Name</th>
+              <th>Type</th>
+              <th>Behavior</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr id="def-constraint-width">
+              <td><dfn>width</dfn></td>
+              <td><code><a>ConstrainULong</a></code></td>
+              <td>The width or width range, in pixels. As a capability, max MUST
+              reflect the <a>display surface</a>'s width, and min MUST reflect
+              the width of the smallest aspect-preserving representation
+              available through downscaling by the user agent.</td>
+            </tr>
+            <tr id="def-constraint-height">
+              <td><dfn>height</dfn></td>
+              <td><code><a>ConstrainULong</a></code></td>
+              <td>The height or height range, in pixels. As a capability, max
+              MUST reflect the <a>display surface</a>'s height, and min MUST
+              reflect the height of the smallest aspect-preserving
+              representation available through downscaling by the user agent.
+              </td>
+            </tr>
+            <tr id="def-constraint-frameRate">
+              <td><dfn>frameRate</dfn></td>
+              <td><code><a>ConstrainDouble</a></code></td>
+              <td>The frame rate (frames per second) or frame rate range. As a
+              capability, max MUST reflect the <a>display surface</a>'s frame
+              rate, and min MUST reflect the lowest frame rate available through
+              frame decimation by the user agent.</td>
+            </tr>
+            <tr id="def-constraint-aspect">
+              <td><dfn>aspectRatio</dfn></td>
```

Comment:
If you try
```
await track.applyConstraints({aspectRatio: {max: 0.2})
```
it will fail with `OverconstrainedError`. No black bars. This is explained in the downscaling section (there is no "overconstrained" condition. Let me know if it's not sufficient.

Reply:
I've elaborated on the "overconstrained" situation, PTAL.